index.js

```diff
@@ -57,22 +60,25 @@ function counter2() {
 
 /* ⚾️⚾️⚾️ Task 2: inning() ⚾️⚾️⚾️
 Use the inning function below to do the following:
-  1. Return a random whole number of points between 0 and 2 scored by one team in an inning
+  1. Return a random whole number of points between 0 and 2 scored by one team in an inning.
   
   For example: invoking inning() should return a numerical score value of 0, 1, or 2
   
 NOTE: This will be a callback function for the tasks below
 */
 
-function inning(/*Code Here*/){
-    /*Code Here*/
+function inning(){
+  const randScore = Math.floor(Math.random() * 3)
+  return randScore
```

Comment:
Awesome job. Small edit: since you are returning the value of randScore without changing it after declaring the variable you can omit the declaration of the variable. This way you won't be using computer space/power to declare a variable and hold the value when you're only returning the value 

It would look like 
`function inning(){
     return Math.floor(Math.random() * 3)
}`